discovery-infra/test_infra/controllers/node_controllers/node.py

```diff
@@ -98,3 +98,15 @@ def set_boot_order_flow(self, cd_first=False, start=True):
         self.set_boot_order(cd_first)
         if start:
             self.start()
+
+    def get_host_id(self):
+        return self.node_controller.get_host_id(self.name)
+
+    def get_cpu_cores(self):
+        return self.node_controller.get_cpu_cores(self.name)
+
+    def set_cpu_cores(self, core_count):
+        self.node_controller.set_cpu_cores(self.name, core_count)
+
+    def set_ram_kib(self, ram_kib):
```

Comment:
kib meaning 1000 bytes? maybe add docstring so people will know definitely the units

Reply:
No, KiB is 1024 - I put the units into the variable name so that it will be clear